index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="styles.css ">
+        <script src="https://kit.fontawesome.com/5116129a4d.js"
+            crossorigin="anonymous"></script>
+        <title>AALA</title>
+    </head>
+
+    <body>
+
+        <section class="sec1">
+            <div class="desktopdarkheader">
+                <div class="dark-items">
+                    <ul>
+                        <li><i class="fa-brands fa-facebook-f"></i></li>
+                        <li><img src="images/twitter.png" alt="twitter image"></li>
+                        <li>English</li>
+                        <li>My Page</li>
+                        <li>Logout</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="desktoplighterheader">
+                <div class="desktopheader">AALA 2022</div>
+                <nav class="references">
+                    <a href="about.html" class="about">About</a>
+                    <a href="#mainprogramme" class="programme">Program</a>
+                    <a href="">Join</a>
+                    <a href="#sec4">Sponsor</a>
+                    <a href="">News</a>
+                </nav>
+            </div>
```

Comment:
Kindly fix the `font-family` here 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/191007458-ed747e40-c61f-4347-b3f5-546f026bd175.png)

Template design:

![image](https://user-images.githubusercontent.com/73714615/191007572-2bc9373a-f2d8-4bde-9b41-3f1840465ad3.png)

Reply:
lato added, thanks